src/styles/_catalog.scss

```diff
@@ -14,9 +14,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
-  }
+    transition: transform 300ms, color 300ms;
+
+    &:not(:nth-child(4n)) {
+      margin-right: 48px;
+    }
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+    &:hover {
+      transform: scale(1.2);
+      color: #34568b;
```

Comment:
You can variables with colors.

Reply:
yes I know but I use this color only once in this task